File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ComboBox/Appearance/DropDownWidthPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.ComboBox.Appearance
{
    public partial class DropDownWidthPage : UserControl
    {
        public DropDownWidthPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles the Load event of the DropDownWidthPage control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void DropDownWidthPage_Load(object sender, EventArgs e)
        {
            //Nuds value initialization
            mobjNumericUpDown.Value = mobjComboBox.DropDownWidth;
        }

        /// <summary>
        /// Handles the ValueChanged event of the mobjNumericUpDown control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjNumericUpDown_ValueChanged(object sender, EventArgs e)
        {
            //Sets DropDownWidth using NUD's value
            mobjComboBox.DropDownWidth = (int)mobjNumericUpDown.Value;
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ComboBox/Appearance/DropDownWidthPage.vb
Namespace CompanionKit.Controls.ComboBox.Appearance

	Public Class DropDownWidthPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()

		End Sub

        ''' <summary>
        ''' Handles the Load event of the DropDownWidthPage control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        Private Sub DropDownWidthPage_Load(sender As Object, e As EventArgs) Handles Me.Load
            'Nuds value initialization
            mobjNumericUpDown.Value = mobjComboBox.DropDownWidth
        End Sub

        ''' <summary>
        ''' Handles the ValueChanged event of the mobjNumericUpDown control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        Private Sub mobjNumericUpDown_ValueChanged(sender As Object, e As EventArgs) Handles mobjNumericUpDown.ValueChanged
            'Sets DropDownWidth using NUD's value
            mobjComboBox.DropDownWidth = CInt(mobjNumericUpDown.Value)
        End Sub
    End Class

End Namespace